art/functions.py

```diff
@@ -154,7 +160,24 @@ def tprint(
             sep=sep,
             space=space,
             __detailed_return=True)
-        print(result)
+        if mode == "line":
+            for line in result.split("\n"):
+                print(line, flush=True)
+                time.sleep(delay)
+        elif mode == "char":
```

Comment:
This section(`char`) is a bit weird! 
Why didn’t you use something like this:

```python
for i in result:
    print(i, end="")
    time.sleep(delay)
```

Reply:
I thought that was not what we wanted, this will print the literal characters, I thought we need the projected characters to be printed (multi-line `a` instead of one char `a`). I can change it if you think that's better.